src/main/java/com/manoelcampos/javadoc/coverage/CoverageDoclet.java

```diff
@@ -75,16 +78,23 @@ public static boolean start(final RootDoc rootDoc) {
      */
     public CoverageDoclet(final RootDoc rootDoc) {
         this.rootDoc = rootDoc;
+        computeOnlyForPublicModifier = "public".equals(getOptionValue(COVERAGE_ONLY_FOR_PUBLIC_OPTION));
+
+        // this needs to be the last part as it already accesses some stuff from the doclet
         this.exporter = new HtmlExporter(this);
     }
 
+    public final boolean computeOnlyForPublicModifier() {
+        return computeOnlyForPublicModifier;
+    }
+
     /**
      * Checks if a given parameter is a valid custom parameter accepted by this doclet.
      * @param paramName the name of the parameter to check
      * @return true if it's a valid custom parameter, false otherwise
      */
     private static boolean isCustomParameter(final String paramName) {
-        return isParameter(paramName, OUTPUT_NAME_OPTION);
+        return isParameter(paramName, OUTPUT_NAME_OPTION) || isParameter(paramName, COVERAGE_ONLY_FOR_PUBLIC_OPTION);
```

Comment:
I think that it should be created a new class to define a command line parameter. The class would include the short and long param names and a description. Then, a list of custom parameters could be defined, so that it can be iterated here to avoid changing the method every time a new parameter is included.

Reply:
that was part of the configuration rework